kolibri/core/assets/src/vue/core-base.vue

```diff
@@ -85,7 +85,6 @@
     padding-bottom: 50px
     z-index: -2
     @media (max-width: $medium-breakpoint + 1)
-      padding-left: 69px
```

Comment:
this change seems to introduce breakage in the layout of the manage app.

what was the intent?